app/assistants/classroom_support/co_teacher/metadata.json

```diff
@@ -1,8 +1,18 @@
 {
     "inputs": [
         {
-            "label": "Action",
-            "name": "action",
+            "label": "User's name",
+            "name": "user_name",
+            "type": "text"
+        },
+        {
+            "label": "User's age",
+            "name": "user_age",
+            "type": "number"
+        },
+        {
+            "label": "User's preference",
+            "name": "user_preference",
```

Comment:
Lets make this global. We dont need different inputs per assistant, they'll all be the same.

Reply:
Done